control-plane/csi-driver/src/bin/node/dev/nvmf.rs

```diff
@@ -170,6 +181,7 @@ impl Attach for NvmfAttach {
                 };
                 let ca = ConnectArgsBuilder::default()
                     .traddr(&self.host)
+                    .transport(TrType::from_str(self.transport.as_str()).unwrap_or_default())
```

Comment:
We can impl TryFrom from the enum to TrType

```suggestion
                    .transport(TrType::try_from(self.transport)?)
```

Reply:
> We can impl TryFrom from the enum to TrType

What'd be benefit as compared to `FromStr` ?

Reply:
If we make transport an enum as I suggested then you have to use the type generic `TryFrom`